### PR TITLE
Stabilize the build

### PR DESCRIPTION
**Pull request by thedrow**
*Note*: Before submitting this pull request, please review our [contributing
guidelines](https://docs.celeryproject.org/en/master/contributing.html).

## Description

Both Kombu 4.6.4 and 4.6.5 break our build currently.
We will release a new version of Kombu that deals with the issues we had.
In the meanwhile, we require better visibility into the state of the build in
Celery so we blacklist those versions from our requirements.

### CODE REVIEW

**Comment by thedrow** [comment]
Unfortunately it seems that one of those versions fixed the problem in #5373 which caused these tasks to hang the worker. :/
I still believe we should exclude those versions.
Please allow me to find out which versions don't break our build.

**Comment by thedrow** [comment]
It seems that the entire 4.6.x series causes our build to fail.
I'm going to mark these tests as xfail for now on the condition that Kombu is below 4.6.0.
Later on, we can release a new version and see how we can deal with all of our Redis broker issues.

**Comment by thedrow** [comment]
I'm confused.
The tests pass locally with Kombu 4.5.0.

**Comment by auvipy** [comment]
Strange! I will start investigating all these tomorrow morning.

**Comment by thedrow** [comment]
Nah I just missed some commas :smile:

**Comment by thedrow** [comment]
The build now should represent the current state of Celery better.
Once the Kombu fix lands, we will be able to verify that the Redis broker works with all the other changes we have.

File: t/integration/test_canvas.py [hunk 1]
```diff
@@ -8,6 +8,8 @@
 from celery.exceptions import TimeoutError
 from celery.result import AsyncResult, GroupResult, ResultSet
 
+import kombu
```
[the remark this answers is not in the document]
Comment (thedrow) [reply t/integration/test_canvas.py]: Nope.
I'll remove it.

**Comment by auvipy** [comment]
this can be closed as with kombu 4.6.6 build is stable

**Comment by thedrow** [comment]
Why should it be closed?
The rest of the versions are still buggy.

**Pull request closed without merging**